Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM hmctspublic.azurecr.io/base/node:12-alpine as base
+FROM hmctspublic.azurecr.io/base/node:16-alpine as base
 
-COPY package.json yarn.lock ./
+#COPY package.json yarn.lock ./
```

Comment:
This I will resolve once the overall build is working - It isn't as straight-forward as updating direct dependencies as I'll also need to update some HMCTS packages which themselves have out-of-date dependencies.